secant/UI Components/TextFields/MultiLineTextField/MultiLineTextFieldStore.swift

```diff
@@ -86,5 +86,9 @@ extension MultiLineTextFieldStore {
 // MARK: - Placeholders
 
 extension MultiLineTextFieldState {
-    static let placeholder = MultiLineTextFieldState()
+    static let placeholder = {
```

Comment:
Bitrise is failing and it seems some xcode version Bitrise has can't derive the Type, we probably need to  
```swift
static let placeholder: MultiLineTextFieldState  = {
```